src/elm/Data/GameConfiguration.elm

```diff
@@ -48,6 +50,13 @@ type alias LevelDict a =
     AllDict.AllDict Level a String
 
 
+bimapLevelDict : (Level -> b) -> (a -> c) -> LevelDict a -> List ( b, c )
+bimapLevelDict mapKey mapValue dict =
```

Comment:
FWIW, the Elm equivalent to Haskell's `bimap` is usually called `mapBoth`.

Reply:
Also this is not actually mapping because it does not return a level dict. One of the rules of mapping is that that only the values inside the structure are changed, not the structure. Since a level dict has hardcoded keys, I don't think it makes sense to implement bimap/mapBoth here?

Reply:
Are you suggesting that we find a new implementation or that this should this be called something else?